reactjs-todo-dv/client/router.js

```diff
@@ -0,0 +1,66 @@
+/*
+ * forgerock-sample-web-react
+ *
+ * router.js
+ *
+ * Copyright (c) 2021 ForgeRock. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+import React, { useEffect } from 'react';
+import { BrowserRouter, Routes, Route, useLocation } from 'react-router-dom';
+
+import { ProtectedRoute } from './utilities/route';
+import Todos from './views/todos';
+import Footer from './components/layout/footer';
+import Header from './components/layout/header';
+import Home from './views/home';
+import Login from './views/login';
+import Logout from './views/logout';
+
+function ScrollToTop() {
+  const { pathname } = useLocation();
+
+  useEffect(() => {
+    window.scrollTo(0, 0);
+  }, [pathname]);
+
+  return null;
+}
+
+/**
+ * @function App - Application React view
+ * @returns {Object} - React component object
+ */
+export default function Router() {
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route path="login" element={<Login />} />
```

Comment:
I noticed that the `register` path was removed from the routing. Can we preserve this feature, and have `/register` go straight to a register flow, rather than login and then having to click on register again?

Reply:
I'd suggest that if the "sign up" (register) link is clicked, we explicitly call the register flow through `acr_values`. That way, we immediately load up the flow that the user selected. Thoughts?

Reply:
There is only 1 flow in the current OOTB DaVinci environments that encompasses both login and register.  Might be easier to talk about this one on a call.

Reply:
Okay, if it's more complicated, we can set this aside when we both have a moment to chat. So, don't focus on this atm.